assets/scripts/src/choices.js

```diff
@@ -2348,6 +2347,14 @@ class Choices {
       dropdown.insertBefore(input, dropdown.firstChild);
     }
 
+    if (
+      this.placeholder &&
+      this.passedElement.type !== 'select-one' &&
+      !this.config.searchPlaceholderValue
+    ) {
+      input.style.width = getWidthOfInput(input);
```

Comment:
Interesting, when I tested this I didn't need to use `requestAnimationFrame` nor `setTimeout`. Hmmmm